src/Microsoft.DotNet.Build.Tasks.Packaging/src/GenerateNetStandardSupportTable.cs

```diff
@@ -68,27 +68,22 @@ public override bool Execute()
             SortedSet<Version> knownNetStandardVersions = new SortedSet<Version>();
             List<SupportRow> rows = new List<SupportRow>(Reports.Length);
 
-            foreach (var report in Reports.Select(r => r.GetMetadata("FullPath")))
+            foreach (var reportPath in Reports.Select(r => r.GetMetadata("FullPath")))
```

Comment:
Its just a normal MSBuild item with default metadata, pointing to the package report which is described by that type itself which is not directly serialized/deserialized.  This change actually makes the code much more self-documenting then what was here before.